gui/pulsed/pulsed_maingui.py

```diff
@@ -389,6 +390,11 @@ def _connect_logic_signals(self):
         self.pulsedmasterlogic().sigMeasurementSettingsUpdated.connect(self.measurement_settings_updated)
         self.pulsedmasterlogic().sigAnalysisSettingsUpdated.connect(self.analysis_settings_updated)
         self.pulsedmasterlogic().sigExtractionSettingsUpdated.connect(self.extraction_settings_updated)
+        self.pulsedmasterlogic().sigSampleBlockEnsemble.connect(self.sampling_or_loading_busy)
```

Comment:
Something is wrong in disconnection. Causes error when unloading and reloading pulsed gui.

Reply:
Ok, seems to be unrelated to this PR but an issue in the current master (tested on 2 setups):
In the manager hit the buttons:
- "deactivate module"
- "Load this module and all dependencies" (NOT the reload button)

RuntimeError: wrapped C/C++ object of type QComboBox has been deleted
Traceback (most recent call last):
  File "C:\Users\Setup3-PC\Desktop\qudi\core\module.py", line 98, in wrap_event
    base_event(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\envs\qudi\lib\site-packages\fysom\__init__.py", line 311, in fn
    self.transition()
  File "C:\ProgramData\Anaconda3\envs\qudi\lib\site-packages\fysom\__init__.py", line 306, in _tran
    self._after_event(e)
  File "C:\ProgramData\Anaconda3\envs\qudi\lib\site-packages\fysom\__init__.py", line 338, in _after_event
    return getattr(self, fnname)(e)
  File "C:\ProgramData\Anaconda3\envs\qudi\lib\site-packages\fysom\__init__.py", line 95, in _callback
    return func(obj, *args, **kwargs)
  File "C:\Users\Setup3-PC\Desktop\qudi\core\module.py", line 223, in __load_status_vars_activate
    self.on_activate()
  File "C:\Users\Setup3-PC\Desktop\qudi\gui\pulsed\pulsed_maingui.py", line 193, in on_activate
    self._activate_generator_settings_ui()
  File "C:\Users\Setup3-PC\Desktop\qudi\gui\pulsed\pulsed_maingui.py", line 944, in _activate_generator_settings_ui
    self.pulse_generator_settings_updated(self.pulsedmasterlogic().pulse_generator_settings)
  File "C:\Users\Setup3-PC\Desktop\qudi\gui\pulsed\pulsed_maingui.py", line 1378, in pulse_generator_settings_updated
    widget.blockSignals(True)